lib/engine/src/artifact.rs

```diff
@@ -96,7 +96,7 @@ pub trait Artifact: Send + Sync + Upcastable {
 
         let module = self.module();
         let (instance_ptr, offsets) = InstanceHandle::allocate_instance(&module);
-        let (imports, import_initializers) = {
+        let (imports, import_envs) = {
```

Comment:
I think a var like `import_function_env_initializers` is a bit more clear. Thoughts?

Reply:
hmm, I think `import_envs` makes more sense because a bunch of things called `ImportEnv`, it's more than just the initializers now

Reply:
I'm open to renaming the type though if `ImportEnv` isn't descriptive enough

Reply:
But they are only relevant on the context of a function, right?

Reply:
Yeah, I was more objecting to the `initializers` part, I think `function` is good 👍  saw your other comments, I'll update some names